multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -24,6 +24,10 @@ externalDNSName: {{.ExternalDNSName}}
 # it for you.
 #hostedZone: ""
 
+# The ID of hosted zone to add the externalDNSName to.
+# Only necessary if there are multiple hosted zones with the same name
+#hostedZoneId: ""
```

Comment:
Seems reasonable not to break this configurable, but may be worth commenting `hostedZone` as deprecated and to use the ID instead (if that's our plan) - then at some release point drop the option.